data/rETH/data.json

```diff
@@ -16,6 +16,9 @@
     },
     "optimism-goerli": {
       "address": "0x1010824414d822cd2167F9aD73a2a99bd2e0E4aE"
+    },
+    "base": {
+      "address": "0xB6fe221Fe9EeF5aBa221c348bA20A1Bf5e73624c1"
```

Comment:
This address doesn't seem to exist... typo?

https://basescan.org/search?f=0&q=0xB6fe221Fe9EeF5aBa221c348bA20A1Bf5e73624c1

Reply:
Somehow a trailing 1 got in there whoops. Fixed now thanks.

Reply:
thanks, looks good, though I guess you used a different factory to generate -- (https://basescan.org/address/0x35a94b6319f0ba9ec930e02cb64ec6bb170bd3f1)

does that add anything to the one I suggested?

Reply:
oh never mind looks like you did use it just maybe through another contract call